smac/facade/smac_bo_facade.py

```diff
@@ -1,9 +1,10 @@
 import numpy as np
-import george
+import skopt
```

Comment:
Do we use this import?

Reply:
Yes, we use it together with the mock library (and mock out parts of skopt).

Reply:
No, I removed it again.